hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -675,6 +675,11 @@ public final class OzoneConfigKeys {
   public static final String HDDS_SCM_CLIENT_FAILOVER_MAX_RETRY =
       "hdds.scmclient.failover.max.retry";
 
+  public static final String
+      OZONE_SCM_NETWORK_TOPOLOGY_CLUSTER_TREE_REFRESH_DURATION =
+      "ozone.scm.network.topology.cluster.tree.refresh.duration";
```

Comment:
It's better to call it "ozone.om.network.topology.cluster.tree.refresh.duration" for it's used in OM.

Reply:
Thank you, I have renamed the configuration